.pre-commit-config.yaml

```diff
@@ -13,14 +13,14 @@ repos:
     rev: 23.9.1
     hooks:
       - id: black
-        exclude: "examples|tests|docs"
+        exclude: "examples|tests/assets"
 
   - repo: https://github.com/charliermarsh/ruff-pre-commit
     # Ruff version.
     rev: "v0.0.289"
     hooks:
       - id: ruff
-        exclude: "examples|tests|docs"
```

Comment:
It annoys me a bit that we skip examples and the such as it breaks consistency but this might need a deeper look and follow up

Reply:
I skipped examples because they have config files, which by design use unbound variables.